### PR TITLE
Gk 127 testy 1

### PR DESCRIPTION
**Pull request by sy1wi4**
repositories, JPA queries tests

### CODE REVIEW

File: backend/src/test/java/pl/edu/agh/api/controller/MatchManagementControllerTest.java [hunk 1]
```diff
@@ -8,37 +8,35 @@
...
 
     @Test
-    void getMatches() throws Exception{
+    void getMatches() throws Exception {
         //given
         UUID id = UUID.randomUUID();
-        Match match = new Match(id, null, null, null, null, null,
-                null, null, null, null, null);
+        Match match = new Match();
+        match.setId(id);
```
Comment (ajablonsk1) [inline backend/src/test/java/pl/edu/agh/api/controller/MatchManagementControllerTest.java:39]: ye it looks better this way, i don't know why i forgot that i can do it this way XD

File: backend/src/test/java/pl/edu/agh/api/controller/ActorsControllerTest.java [hunk 2]
```diff
@@ -18,17 +22,14 @@
...
-import java.util.List;
-import java.util.UUID;
-
 
 @WebMvcTest(ActorsController.class)
 public class ActorsControllerTest {
-    @Autowired private MockMvc mockMvc;
-    @MockBean private FighterService fighterService;
     private final ObjectMapper objectMapper = new ObjectMapper();
+    @Autowired
```
Comment (ajablonsk1) [inline backend/src/test/java/pl/edu/agh/api/controller/ActorsControllerTest.java:29]: inline annotations looks clearer to me, but I guess it's subjective

**Review by ajablonsk1 — APPROVED** [review]
Nice!

File: backend/src/test/java/pl/edu/agh/api/controller/ActorsControllerTest.java (hunk 2 above)
Comment (sy1wi4) [reply backend/src/test/java/pl/edu/agh/api/controller/ActorsControllerTest.java:29]: it has been changed by the intellij auto-formatter and tbh I prefer it this way

**Pull request merged**